looper/looper.py

```diff
@@ -684,7 +646,29 @@ def main():
     if args.command == "run":
         if args.compute:
             prj.set_compute(args.compute)
-        run(prj, args, remaining_args)
+
+        # TODO split here, spawning separate run process for each pipelines directory in project metadata pipelines directory.
+        pipedirs = prj.metadata.pipelines_dir
+        _LOGGER.debug("Pipelines dirpath(s): {}".format(pipedirs))
+
+        for pipedir in prj.metadata.pipelines_dir:
```

Comment:
Instead of looping through pipeline directories in looper, the project object itself should know how to deal with them; someone outside of looper may need to handle multiple pipeline directories.

In general we should stick as much logic as we can into models, since it's universal.